spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -298,6 +305,10 @@ public Duration getConnectionTimeout() {
 		return this.connectionTimeout;
 	}
 
+	public int getRequestedChannelMax() {
```

Comment:
Getter/Setter should be co-located.

Reply:
Attempt was made not to mess up existing ordering at the end of RabbitProperties, but I was not able to see this type of code convention, since getConnectionTimeout and setConnectionTimout are not co-located. Updated PR with changes